### PR TITLE
[Cards] Add missing Shapes dependency.

### PR DESCRIPTION
**Pull request by jverkoey**
https://github.com/material-components/material-components-ios/commit/af2a3c89d2db301ad05fd2d3eb1e6dae5f057e46 updated the BUILD deps but not the CocoaPods dependencies.

### CODE REVIEW

File: MaterialComponents.podspec [hunk 1]
```diff
@@ -211,6 +211,7 @@ Pod::Spec.new do |mdc|
     component.dependency "MaterialComponents/ShadowLayer"
     component.dependency "MaterialComponents/private/Icons/ic_check_circle"
     component.dependency "MaterialComponents/private/Math"
+    component.dependency "MaterialComponents/private/Shapes"
```
Comment (romoore) [inline MaterialComponents.podspec:214]: The summary doesn't have much context - why is this needed? I don't see any code changes.

Comment (jverkoey) [reply MaterialComponents.podspec:214]: Updated the PR description.
